File: Competetive programming/C++/Unit 3/File_Fragmentation.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

    string fragment1, fragment2;

    cout << "===== FILE FRAGMENTATION =====\n\n";

    cout << "Enter First Fragment:\n";
    cout << "Example: HELLO\n";
    cin >> fragment1;

    cout << "\nEnter Second Fragment:\n";
    cout << "Example: WORLD\n";
    cin >> fragment2;

    string original = fragment1 + fragment2;

    cout << "\nOriginal File Reconstructed:\n";
    cout << original << endl;

    return 0;
}

File: Competetive programming/Python/Unit 3/File_Fragmentation.py
print("===== FILE FRAGMENTATION =====\n")

n = int(input("Enter number of fragments: "))

fragments = []

for i in range(n):

    fragments.append(input(f"Fragment {i+1}: "))

print("\nReconstructed File:")

result = ""

for item in fragments:
    result += item

print(result)